client/src/views/Models/Model.vue

```diff
@@ -351,10 +352,17 @@
       this.selectedLayoutId = layoutId;
     }
 
-    async searchCosmos (keyword: string): Promise<void> {
+    async getDrilldownKnowledge (): Promise<void> {
       try {
-        const response = await cosmosSearch(filterToParamObj({ cosmosQuery: [keyword] }));
-        this.drilldownKnowledge = response;
+        if (this.selectedGraphMetadata?.constructor === Array) {
```

Comment:
What happens if there is no metadata for a node? I think we used to have an error message informing about this. I think it looks weird and not informative when the panel is just empty.
![image](https://user-images.githubusercontent.com/10552785/127848210-318e5376-32a8-462c-82f8-5d0a2312bdab.png)

Reply:
There is a message, using the new component you created.
I think you stumble onto a bug from `main`.

I fixed it, so it should display the message as you expected:
![Screen Shot 2021-08-02 at 20 04 28](https://user-images.githubusercontent.com/636801/127938126-28863f7b-3c34-4a3f-8c3d-c4634b8f7a8e.png)